azure-pipelines.yml

```diff
@@ -346,9 +346,9 @@ stages:
               vs_release:
                 _configuration: Release
                 _testKind: testVs
-              inttests_release:
-                _configuration: Release
-                _testKind: testIntegration
+              # inttests_release:
```

Comment:
Can you just remove these rather than comment them out.

Reply:
There you go